pandas/tests/extension/base/reshaping.py

```diff
@@ -62,11 +62,11 @@ def test_concat_mixed_dtypes(self, data):
         self.assert_series_equal(result, expected)
 
         # simple test for just EA and one other
-        result = pd.concat([df1, df2])
+        result = pd.concat([df1, df2.astype(object)])
```

Comment:
changing to object dtype here, as that is a more robust base test (concatting anything with object should always result in object, generally). The current `df2` is numpy int, which makes this test invalid for the nullable Int64 extension dtype